packages/trie/test/trie/trie.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { Level } from 'level'
+import { MemoryLevel } from 'memory-level'
+import { tmpdir } from 'os'
+import * as tape from 'tape'
+import { LevelDB, ROOT_DB_KEY, Trie } from '../../src'
+
+tape('Trie', function (t) {
+  t.test(
+    'creates an instance via the static constructor `make` function and defaults to `true` with a database',
```

Comment:
Can you update all these test comments to reference `create` instead of `make`? After that, I'll approve and we can get this merged. Great work!